ajax/libs/react-widgets/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "react-widgets",
+  "description": "An à la carte set of polished, extensible, and accessible inputs built for React",
+  "filename": "react-widgets.min.js",
+  "author": {
+    "name": "Jason",
+    "email": "monastic.panic@gmail.com"
+  },
+  "keywords": [
+    "react",
+    "widgets",
+    "dropdown",
+    "combobox",
+    "calendar",
+    "datepicker",
+    "date picker",
+    "numberpicker",
+    "number picker",
+    "radio group",
+    "checkbox list",
+    "multiselect",
+    "form",
+    "input",
+    "react-ui",
+    "react-component"
+  ],
+  "homepage": "http://jquense.github.io/react-widgets/docs/",
```

Comment:
Let's use `https` https://jquense.github.io/react-widgets/docs/
Using https is better because it's more security for browsing. to prevent the possibility the users been attacked during the redirection.
Thank you.